<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>677</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Apliakasi kommas</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/lock-26.png</normaloff>:/lock-26.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QMdiArea" name="mdiArea"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>677</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionTambah_enkrpsi_pesan"/>
    <addaction name="actionTambah_dekripsi_pesan"/>
    <addaction name="actionKeluar"/>
   </widget>
   <widget class="QMenu" name="menuTentang">
    <property name="title">
     <string>History</string>
    </property>
    <addaction name="actionDaftarPesanEnkripsi"/>
    <addaction name="actionDaftarPesanDekripsi"/>
    <addaction name="actionHapus_history"/>
   </widget>
   <widget class="QMenu" name="menuTentang_2">
    <property name="title">
     <string>Bantuan</string>
    </property>
    <addaction name="actionTentang"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTentang"/>
   <addaction name="menuTentang_2"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionTambah_enkrpsi_pesan">
   <property name="text">
    <string>Tambah enkrpsi pesan</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionDaftarPesanEnkripsi">
   <property name="text">
    <string>Pesan enkripsi</string>
   </property>
   <property name="shortcut">
    <string>Shift+E</string>
   </property>
  </action>
  <action name="actionTambah_dekripsi_pesan">
   <property name="text">
    <string>Tambah dekripsi pesan</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionDaftarPesanDekripsi">
   <property name="text">
    <string>Pesan dekripsi</string>
   </property>
   <property name="shortcut">
    <string>Shift+D</string>
   </property>
  </action>
  <action name="actionTentang">
   <property name="text">
    <string>Tentang</string>
   </property>
  </action>
  <action name="actionKeluar">
   <property name="text">
    <string>Keluar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionHapus_history">
   <property name="text">
    <string>Hapus semua history</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
